Mathlib/Algebra/Category/BialgebraCat/Monoidal.lean

```diff
@@ -61,8 +61,8 @@ noncomputable instance instMonoidalCategory : MonoidalCategory (BialgebraCat R)
 /-- `forget₂ (BialgebraCat R) (AlgebraCat R)` as a monoidal functor. -/
 noncomputable instance : (forget₂ (BialgebraCat R) (AlgebraCat R)).Monoidal where
 
-#adaptation_note /-- nightly-2025-02-18: added `maxHeartbeats` --/
 set_option maxHeartbeats 400000 in
+-- nightly-2025-02-18: added `maxHeartbeats`
```

Comment:
This is not ideal - but since `set_option maxHeartbeats` is also tracked as technical debt, I guess this is fine.